interceptor.go

```diff
@@ -247,7 +245,13 @@ func (i *Interceptor) WrapStreamingClient(next connect.StreamingClientFunc) conn
 				span.SetAttributes(headerAttributes(protocol, responseKey, conn.ResponseHeader(), i.config.responseHeaderKeys)...)
 				span.SetStatus(spanStatus(protocol, state.error))
 				span.End()
-				instrumentation.duration.Record(ctx, i.config.now().Sub(requestStartTime).Milliseconds(), metric.WithAttributes(state.attributes...))
+				instrumentation.requestsPerRPC.Record(ctx, state.sentCounter,
```

Comment:
Importantly `requests` are `sendCounter` here and below for the handler the reverse.